WORKSPACE

```diff
@@ -43,6 +43,46 @@ container_load(
     file = "//testdata:pause.tar",
 )
 
+container_pull(
+    name = "alpine_linux_amd64",
+    registry = "index.docker.io",
+    repository = "library/alpine",
+    tag = "3.8",
+)
+
+container_pull(
+    name = "alpine_linux_armv6",
+    architecture = "arm",
+    cpu_variant = "v6",
+    os = "linux",
+    registry = "index.docker.io",
+    repository = "library/alpine",
+    tag = "3.8",
+)
+
+container_pull(
+    name = "alpine_linux_ppc64le",
+    architecture = "ppc64le",
+    registry = "index.docker.io",
+    repository = "library/alpine",
+    tag = "3.8",
+)
+
+container_pull(
+    name = "k8s_pause_amd64",
+    registry = "k8s.gcr.io",
+    repository = "pause",
+    tag = "3.1",
+)
+
+container_pull(
+    name = "k8s_pause_arm64",
+    architecture = "arm64",
+    registry = "k8s.gcr.io",
+    repository = "pause",
```

Comment:
same here, add sha?

Reply:
I used the kubernetes pause images here because
a) they're small
b) they're unlikely to be updated
c) if we do update them, we have a policy of not overwriting existing tags

I guess I'll have to eat my hat if this changes. :)

Reply:
sgtm

Reply:
sgtm